src/Database/Statement/BufferedStatement.php

```diff
@@ -92,12 +93,14 @@ public function __construct(StatementInterface $statement, DriverInterface $driv
      * @param string $property internal property to get
      * @return mixed
      */
-    public function __get(string $property)
+    public function __get(string $property): mixed
     {
         if ($property === 'queryString') {
             /** @psalm-suppress NoInterfaceProperties */
             return $this->statement->queryString;
         }
+
+        throw new RuntimeException("Cannot access undefined property `$property`.");
```

Comment:
This could be `InvalidArgumentException` if you wanted a more accurate exception type.